cogs5e/models/automation/effects/attack.py

```diff
@@ -153,7 +153,7 @@ def run(self, autoctx):
             # assign hit values
             if d20_value >= criton or to_hit_roll.crit == d20.CritType.CRIT:  # natural crit
                 did_crit = True if not nocrit else False
-            elif to_hit_roll.crit == d20.CritType.FAIL:  # crit fail
+            elif to_hit_roll.crit == d20.CritType.FAIL or force_roll == 1:  # crit fail
```

Comment:
```suggestion
            elif d20_value == 1 or to_hit_roll.crit == d20.CritType.FAIL:  # crit fail
```
Just to match how the crit branch is laid out above. `d20_value` should be set to the force roll if supplied since it's defined as `d20_value = d20.utils.leftmost(to_hit_roll.expr).total`